industrial_taxonomy/pipeline/glass_house/flow.py

```diff
@@ -0,0 +1,196 @@
+from typing import List
+
+from metaflow import (
+    conda_base,
+    current,
+    FlowSpec,
+    Parameter,
+    resources,
+    pip,
+    project,
+    Run,
+    step,
+    S3,
+)
+from metaflow.datatools.s3 import MetaflowS3NotFound
+
+from industrial_taxonomy.getters.glass_house import FullMatchResult
+
+try:  # Hack for type-hints on attributes
+    from pandas import DataFrame
+except ImportError:
+    pass
+
+
+@conda_base(python="3.9")  # Need >=3.8 on Batch for TypedDict of FullMatchResult
+@project(name="industrial_taxonomy")
+class JacchammerFlow(FlowSpec):
+    """Fuzzy match Companies House and Glass AI names using Jacchammer.
+
+    Attributes:
+        run_id: The origin run ID of the flow
+        names_x: Companies House company names
+        names_y: Glass AI organisation names
+        clean_names_x: Pre-processed Companies House company names
+        clean_names_y: Pre-processed Glass AI organisation names
+        full_top_matches: Full match results
+    """
+
+    test_mode = Parameter(
+        "test-mode",
+        help="Whether to run in test mode (on a small subset of data)",
+        type=bool,
+        default=lambda _: not current.is_production,
+    )
+    should_clean_names = Parameter(
+        "clean-names",
+        help="Whether to clean names using Jacchammer default `preproc_names`",
+        type=bool,
+        default=False,
+    )
+
+    names_x: "DataFrame"
+    names_y: "DataFrame"
+    clean_names_x: "DataFrame"
+    clean_names_y: "DataFrame"
+    run_id: int
+    full_top_matches: List[FullMatchResult]  # TODO: py37 error
+
+    @step
+    def start(self):
+        """Load raw data"""
+        from industrial_taxonomy.getters.companies_house import (
+            get_name as get_companies_house_names,
```

Comment:
Point for clarification / minor: Would it make more sense to give these getters more informative names instead of renaming them during the import?

Reply:
Because e.g. `get_name` is in a module called `companies_house` then it doesn't make much sense to name the function `get_companies_house_name`.

But you are right in how I've done these imports  isn't right.

I think the best of both worlds would be the following...

```python
from industrial_taxonomy.getters import glass, companies_house 
glass.get_organisation()
companies_house.get_name()
```